sample_tpl/ind_cte_scan_temp_table_insert.tpl

```diff
@@ -0,0 +1,62 @@
+// Test that Accumulate() returns false on empty
+// Returns the number of tuples returned (should be 0)
+
+fun main(exec_ctx: *ExecutionContext) -> int32 {
+  // Initialize CTE Scan Iterator
+
+  var col_oids: [1]uint32
+  col_oids[0] = 1
+  var col_types: [1]uint32
+  col_types[0] = 4
```

Comment:
Can you use the testCatalogLookup builtin? The intent is to prevent strange .tpl file breakage when test tables are edited.